unite_contranges.py

```diff
@@ -0,0 +1,146 @@
+#Allison Towner
+#09 October 2020
+
+#continuum_selections.py in analysis code is located at:
+#in ALMA_IMF/reduction/analysis/
+
+def merge_contdotdat(field='',band=''):
+
+	import numpy as np
+	import sys
+	sys.path.append('.')
+	from parse_contdotdat import parse_contdotdat
+
+	#Read in metadata from .json file
+	basepath = '/orange/adamginsburg/ALMA_IMF/2017.1.01355.L'
+	with open(basepath + '/contdatfiles.json', 'r') as fh:
+	    contdatfiles = json.load(fh)
+	with open(basepath + '/metadata.json', 'r') as fh:
+	    metadata = json.load(fh)
+
+
+	##Set field and band name - this should be detected automatically in the final version of the script but is hardwired here for testing purposes
+	#field = 'G327.29'
+	#band = 'B3'
+
+	#use parse_contdotdat to read the cont.dat files from their respective directories. Should be flexible enough to account for missing configurations
+	contdotdat_ranges = []
+	for i,m in enumerate(metadata[band][field]['cont.dat']):
+  		m = str(m)
+    	contdotdat_ranges.append(parse_contdotdat(metadata[band][field]['cont.dat'][m]))
```

Comment:
it looks like you lost an indentation here?  is `m` supposed to be the cont.dat filename?

Reply:
yep, that's what happened